src/screens/SignInV2/index.js

```diff
@@ -179,7 +179,7 @@ const S = StyleSheet.create({
     paddingTop: 26
   },
   btn: {
-    backgroundColor: '#023B60',
+    backgroundColor: COLORS.blueLoyal,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from a hardcoded color value to a reference from the `COLORS` object is a good practice for maintainability and consistency. This allows for easier updates in the future if the color scheme changes.
```diff
-    backgroundColor: '#023B60',
+    backgroundColor: COLORS.blueLoyal,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->